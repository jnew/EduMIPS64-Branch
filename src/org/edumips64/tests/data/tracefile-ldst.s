; Test for the Dinero tracefile generation code.
;
; Will produce a tracefile with 3 instruction accesses, 1 load and 1 store.
.data
src: .word 42
dst: .word 0

.code
ld r1,src(r0)
sd r1,dst(r0)
syscall 0
