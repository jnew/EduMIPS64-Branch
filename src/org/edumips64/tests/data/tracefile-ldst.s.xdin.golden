i 0000000000000000 4
i 0000000000000004 4
r 0000000000000010 8
i 0000000000000008 4
w 0000000000000018 8
